src/frontends/sql/decode.cc

```diff
@@ -74,12 +74,14 @@ static Value GetExprValue(const Expression &expr,
   CHECK(false) << "Unreachable!";
 }
 
-const Value &DecodeExpression(const Expression &expr,
-                              DecoderContext &decoder_context) {
+Value DecodeExpression(const Expression &expr,
```

Comment:
Is memory safety the reason for this change?

Reply:
It's more about the fact that `NamedValueMap` has value-typed `Value`s as its value and that `Value` is pretty small and naively-copyable anyway.